src/pretix/base/settings.py

```diff
@@ -2261,6 +2262,18 @@ def unserialize(cls, s):
                         'many years. If you keep it empty, gift cards do not have an explicit expiry date.'),
         )
     },
+    'privacy_policy': {
+        'default': None,
+        'type': LazyI18nString,
+        'form_class': I18nURLFormField,
+        'form_kwargs': dict(
+            label=_("Privacy Policy URL"),
+            help_text=_("This should point e.g. to a part of your website that explains how you use data gathered in "
+                        "your ticket shop."),
```

Comment:
Change text to: This should link to a section of your website that explains how you use the data gathered in your ticket shop.

Reply:
Hi @mariobehling, changed